### PR TITLE
Add details of Model panels

### PR DESCRIPTION
**Pull request by DavidePrincipi**
- [x] Describe the provisioning options: what the do and how to use them. Add a manual section for each UI panel:
  image
- [x] Add a new provisioning parameters reference page, that describes each parameter from the UI point of view
- [x] Describe the soft and line keys types

https://github.com/nethesis/dev/issues/5833

### CODE REVIEW

File: admin-manual/wizard2_provisioning.rst [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+
+=========================================
+Guida ai parametri di provisioning (beta)
+=========================================
+
+.. warning::
+    
+    Le funzioni descritte in questa sezione sono disponibili in anteprima solo
+    ai membri del |product| Quality Team. Fare riferimento a
+    :ref:`wizard-section` per la versione stabile.
```
Comment (gsanchietti) [inline admin-manual/wizard2_provisioning.rst:11]:
```suggestion
.. warning::
```

File: admin-manual/wizard2_provisioning.rst [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+
+.. _panel-softkeys:
+
+Soft key
+========
+
+I *soft key* sono tasti del telefono programmabili specifici per
+richiamare delle funzioni del telefono.
+
+Qualora il telefono renda disponibili più tasti di quanti ne sono visualizzati
```
Comment (gsanchietti) [inline admin-manual/wizard2_provisioning.rst:32]:
```suggestion
Qualora il telefono renda disponibili più tasti di quanti ne siano visualizzati
```

File: admin-manual/wizard2_provisioning.rst [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+      - No
+      - Sì (predefinita)
+
+.. _panel-linekeys:
+
+Line key
+========
+
+I *line key* sono tasti del telefono programmabili simili ai *soft key* ma
+più specifici per la gestione delle chiamate e il monitoraggio degli altri interni.
```
Comment (gsanchietti) [inline admin-manual/wizard2_provisioning.rst:118]:
```suggestion
più specifici per la gestione delle chiamate e il monitoraggio dello stato degli interni.
```

File: admin-manual/wizard2_provisioning.rst [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+
+Qualora il telefono renda disponibili più tasti di quanti ne sono visualizzati
```
Comment (gsanchietti) [inline admin-manual/wizard2_provisioning.rst:120]:
```suggestion
Qualora il telefono renda disponibili più tasti di quanti ne siano visualizzati
```

File: admin-manual/wizard2_provisioning.rst [hunk 5]
```diff
@@ -0,0 +1,391 @@
...
+
+.. _panel-expkeys:
+
+Exp key
+=======
+
+Gli *expansion key* sono i tasti programmabili presenti sui *moduli di espansione*,
+dispositivi collegabili al telefono che ne aumentano la quantità di tasti disponibili.
+
+Qualora il modulo di espansione renda disponibili più tasti di quanti ne sono visualizzati
```
Comment (gsanchietti) [inline admin-manual/wizard2_provisioning.rst:258]:
```suggestion
Qualora il modulo di espansione renda disponibili più tasti di quanti ne siano visualizzati
```

File: admin-manual/wizard2_provisioning.rst [hunk 6]
```diff
@@ -0,0 +1,391 @@
...
+Questo tipo di tasti si configura come i :ref:`panel-linekeys`.
+
+.. _panel-display:
+
+Schermo e suoneria
+==================
+
+* :guilabel:`Selezione suoneria` Ogni telefono ha alcune suonerie predefinite che possono essere
+  selezionate in base al numero progressivo. Laddove supportata è possibile scegliere la suoneria
+  personalizzata, che va poi caricata nel controllo descritto di seguito.
```
Comment (gsanchietti) [inline admin-manual/wizard2_provisioning.rst:271]:
```suggestion
  personalizzata, che va poi caricata nel campo descritto di seguito.
```

File: admin-manual/wizard2_provisioning.rst [hunk 7]
```diff
@@ -0,0 +1,391 @@
...
+
+* :guilabel:`Gestione suoneria personalizzata` Seleziona un file audio per la suoneria personalizzata
+  caricato in precedenza, o ne carica uno nuovo aprendo l'apposito modulo di gestione. Il formato
+  audio deve essere compatibile con le specifiche del produttore del telefono.
```
Comment (gsanchietti) [inline admin-manual/wizard2_provisioning.rst:275]: Is there any link we can add here?

File: admin-manual/wizard2_provisioning.rst [hunk 8]
```diff
@@ -0,0 +1,391 @@
...
+
+* :guilabel:`Immagine di sfondo` :guilabel:`Immagine screensaver` Seleziona un file immagine
+  rispettivamente per lo sfondo dello schermo del telefono e per lo screensaver, oppure ne carica
+  una nuova aprendo l'apposito pannello di gestione. Il formato immagine deve
+  essere compatibile con le specifiche del produttore del telefono.
```
Comment (gsanchietti) [inline admin-manual/wizard2_provisioning.rst:280]: Like a above: a link to specifications could be helpefult

File: admin-manual/wizard2_provisioning.rst [hunk 9]
```diff
@@ -0,0 +1,391 @@
...
+.. _panel-preferences:
+
+Preferenze
+==========
+
+* :guilabel:`Indirizzo server NTP` Il nome host o l'indirizzo IP del server 
+  NTP (Network Time Protocol) per impostare automaticamente e correttamente l'orario del telefono.
+
+* :guilabel:`Pianificazione del provisioning` Selezionando **Solo all'avvio** i telefoni
+  rinnovano la propria configurazione dopo l'accensione o il riavvio. Invece Selezionando
```
Comment (gsanchietti) [inline admin-manual/wizard2_provisioning.rst:299]:
```suggestion
  rinnovano la propria configurazione dopo l'accensione o il riavvio. Invece selezionando
```

File: admin-manual/wizard2_provisioning.rst (hunk 1 above)
Comment (DavidePrincipi) [reply admin-manual/wizard2_provisioning.rst:11]: I'm swapping the beta/old pages in a further PR

File: admin-manual/wizard2_provisioning.rst (hunk 7 above)
Comment (DavidePrincipi) [reply admin-manual/wizard2_provisioning.rst:275]: No ATM

**Pull request merged**